### PR TITLE
Orders and products table changes..

### PR DESCRIPTION
**Pull request by jasonbringle**
Added orders table.
Renamed Items to products
Renamed stores-orders-products-tags migration. 
Updated migrations to match our schema
Added seed data for new tables

### CODE REVIEW

**Review by Brodt7258 — APPROVED** [review]
looks good. happy to see our new schema taking shape.

File: data/migrations/20201109234734_stores-orders-products-tags.js [hunk 1]
```diff
@@ -17,6 +17,29 @@ exports.up = function (knex) {
...
+      tb.string('delivery_method').notNullable();
+      tb.string('delivery_service');
+      tb.integer('shipping_cost');
+      tb.string('tracking_number');
+      tb.string('status');
+      tb.date('created_at');
+    })
     .createTable('tag', (tb) => {
       tb.increments();
       tb.string('tag_name', 255);
```
Comment (SebastianGarces) [inline data/migrations/20201109234734_stores-orders-products-tags.js]: lets change this property to just name - we want consistency - on the products table we say "name" only but here "tag_name"

File: data/migrations/20201109234734_stores-orders-products-tags.js [hunk 2]
```diff
@@ -17,6 +17,29 @@ exports.up = function (knex) {
...
+        .onUpdate('CASCADE');
+      tb.string('ship_to');
+      tb.string('delivery_method').notNullable();
+      tb.string('delivery_service');
+      tb.integer('shipping_cost');
+      tb.string('tracking_number');
+      tb.string('status');
+      tb.date('created_at');
+    })
     .createTable('tag', (tb) => {
```
Comment (SebastianGarces) [inline data/migrations/20201109234734_stores-orders-products-tags.js]: lets change the table name to "tags"

File: data/migrations/20201109234734_stores-orders-products-tags.js [hunk 3]
```diff
@@ -35,52 +35,69 @@ exports.up = function (knex) {
...
       tb.date('created_at');
     })
-    .createTable('tag', (tb) => {
+    .createTable('tags', (tb) => {
       tb.increments();
       tb.string('tag_name', 255);
+      tb.date('created_at');
     })
-    .createTable('item', (tb) => {
+    .createTable('products', (tb) => {
```
Comment (SebastianGarces) [inline data/migrations/20201109234734_stores-orders-products-tags.js:48]: lets add a store_id and reference the stores table

**Review by SebastianGarces — CHANGES_REQUESTED** [review]
Requested a couple of changes. See comments below.

File: data/migrations/20201109234734_stores-orders-products-tags.js (hunk 2 above)
Comment (SebastianGarces) [reply data/migrations/20201109234734_stores-orders-products-tags.js]: I see this was already changed. 👍

File: data/seeds/04_tags.js [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
   // Inserts seed entries
-  return knex('tag').insert([
-    { tag_name: 'junk' },
-    { tag_name: 'clothes' },
-    { tag_name: 'poor quality' },
-    { tag_name: 'good quality' },
-    { tag_name: 'media' },
-    { tag_name: 'space' },
+  return knex('tags').insert([
+    { tag_name: 'junk', created_at: date.past() },
```
Comment (SebastianGarces) [inline data/seeds/04_tags.js]: Make sure change seeds after you change the property to "name"

**Review by SebastianGarces — APPROVED** [review]
Looks great, tons of tables got done - nice to see our plan coming together

**Pull request merged**